### DISCUSSION TITLE
Festival kit/resource

### DISCUSSION THREAD

**Discussion by TinaJAZ** [resbazaz/festival2024#5] [Q&A] [answered]
Is there be a “festival kit”/resource: 

- Blurbs (i.e. join to help plan, we’re planning, save the date, register, and we’ll see you tomorrow)
- Official logo
- URL’s
- Hashtags
- List of where/to whom to send info to spread the word
- Fact sheet/official details of the festival (still planning, so this will be incomplete for now). 

Who do I work with to help put this kit together?

**Comment by lwdozal** [comment, marked as answer]
Oh I like this idea. Maybe we can create a folder here?

**Reply by lwdozal** [reply]
ok, just created one, we can use it as needed: FestivalKit_resources

**Reply by julianpistorius** [reply]
Good idea @TinaJAZ! Thank you for suggesting it. @lwdozal is the best person to work with.

**Comment by lwdozal** [comment]
I also found this which I think we can work from: https://github.com/resbazaz/resbazaz_promotional_materials
